bin/ffscreencast

```diff
@@ -421,7 +421,7 @@ can_use_ffmpeg() {
 				return "${EXIT_OK}"
 			fi
 		elif [ "${UNAME}" = "Linux" ]; then
-			if ! ${FFMPEG} -version 2>&1 | $GREP '\-\-enable-x11grab' > /dev/null 2>&1; then
+			if ! ${FFMPEG} -devices 2>&1 | $GREP 'x11grab' > /dev/null 2>&1; then
```

Comment:
In order to stay compatible with the older versions, I would suggest to check for both:

```bash
if ! ${FFMPEG} -version 2>&1 | $GREP '\-\-enable-x11grabsd' > /dev/null 2>&1 \
	&& ! ${FFMPEG} -devices 2>&1 | $GREP 'x11grab' > /dev/null 2>&1; then
```